### DISCUSSION TITLE
How to run all test cases locally?

### DISCUSSION THREAD

**Discussion by bykka** [Giorgi/DuckDB.NET#133] [General]
Hi,
I have a problem running all test cases on my local windows PC.
All of those fail with the message "Unable to load DLL 'duckdb' or one of its dependencies: The specified module could not be found. (0x8007007E)"

Could you provide a bit of advice on how to fix that?

![ice_video_20230919-182431(1)](https://github.com/Giorgi/DuckDB.NET/assets/489997/4f580b57-4953-4aac-aaf3-2840ad2c744b)

**Comment by Giorgi** [comment]
@bykka Run `dotnet build /p:BuildType=Full` before running tests. You only need to run it once.
